cljfmt/src/cljfmt/macros.clj

```diff
@@ -0,0 +1,7 @@
+(ns cljfmt.macros
+  (:require [clojure.java.io :as io]))
+
+(defmacro read-resource
```

Comment:
We could do without this file if we just used a different name for the macro. So in the core namespace:

```clojure
#?(:clj (def read-resource (comp read-string slurp io/resource)))
#?(:clj (defmacro read-resource* [path] `'(~read-resource path)))
```

I also think you can avoid the `read-string` and `pr-str` if you quote inside the syntax quote, but I could be wrong about this.

Also, are there any disadvantages in using the macro in *both* Clojure and ClojureScript? Because Clojure compiles top-level forms, it shouldn't matter whether we use `read-resource` or `read-resource*` in a `def`, as far as I can tell...